test/bench.cpp

```diff
@@ -1,90 +1,392 @@
 #include <lorina/bench.hpp>
-#include <cstdio>
+#include <lorina/lorina.hpp>
 #include <fstream>
 #include <string>
 #include <vector>
+#include <stdio.h>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+
+// Define screen dimensions
+#define SCREEN_WIDTH    800
+#define SCREEN_HEIGHT   600
+
+
+#define FONT_PATH   "../assets/ttf/DejaVuSansMono.ttf"
 
 using namespace lorina;
 
-struct bench_statistics
-{
-  uint32_t number_of_inputs = 0;
-  uint32_t number_of_outputs = 0;
-  uint32_t number_of_dffs = 0;
+struct parametrs {
+    std::vector<std::string> inputs;
+    std::string type = NULL;
+    std::string output = NULL;
```

Comment:
Does this even compile? IIRC many C++ compilers use `#define NULL 0` for compatibility reasons, but not all, so the converted value is not even strictly defined. Use empty string instead.